pkgs/development/python-modules/anonip/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, buildPythonPackage, fetchFromGitHub, ipaddress, isPy27 }:
+
+buildPythonPackage rec {
+  pname = "anonip";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "DigitaleGesellschaft";
+    repo = "Anonip";
+    rev = "v${version}";
+    sha256 = "0y5xqivcinp6pwx4whc8ca1n2wxrvff7a2lpbz2dhivilfanmljs";
+  };
+
+  propagatedBuildInputs = lib.optionals isPy27 [ ipaddress ];
+
+  doCheck = true;
```

Comment:
```suggestion
```

Reply:
we're removing it because it's default `true` btw

Reply:
Oh sorry, leftover from testing. Rebased.